<center>
    <img src="https://cf-courses-data.s3.us.cloud-object-storage.appdomain.cloud/IBMDeveloperSkillsNetwork-DS0105EN-SkillsNetwork/labs/Module2/images/SN_web_lightmode.png" width="300" alt="cognitiveclass.ai logo">
</center>


#### Add your code below following the instructions given in the course


DataScienceEcosystem.ipynb

# H1 :Data Science Tools and Ecosystem

In this notebook, Data Science Tools and Ecosystem are summarized.

#  Some of the popular languages that Data Scientists use are:
 
```
  1. Python
  2. Java
  3. C++
  4. C   
```
 

# Some of the commonly used libraries used by Data Scientists include:
 
```
  1.  NumPy
  2.  Pandas
  3.  Matplotlib
  4.  Seaborn
  5.  Scikit-learn
```

| Data Science Tools |
| -------------| 
| Apache Spark | 
| Tableau | 
| Python | 
| Jupyter Notebook| 
| SAS | 
| Excel | 

# H3 :Below are a few examples of evaluating arithmetic expressions in Python.

# This a simple arithmetic expression to mutiply then add integers.




In [8]:
(3*4)+5

17

 # This will convert 200 minutes to hours by diving by 60.

In [15]:
200/60


3.3333333333333335

After executing the above cell, you will find that the output <code>3.33</code> is generated on executing <code>200÷60</code>


# H1: Objectives:
* Python: Widely favored for its extensive libraries (e.g., NumPy, Pandas, Scikit-learn, TensorFlow, PyTorch) and versatility in tasks ranging from data manipulation and analysis to machine learning and deep learning.
* R: A powerful language primarily used for statistical computing and graphics, with a rich ecosystem of packages for statistical modeling, visualization, and specialized analyses.
* SQL (Structured Query Language): Essential for managing and querying data stored in relational databases, a fundamental skill for any data professional.
* Julia: A high-level, high-performance language designed specifically for technical computing and numerical analysis, offering speed comparable to C/C++ while maintaining ease of use.

* Scala: Often used in big data environments, particularly with Apache Spark, for its ability to handle large-scale data processing and its integration with the Java Virtual Machine (JVM).
* Java: A general-purpose language used in data science, especially for building scalable data applications and integrating with existing enterprise systems.


# H3 :Author
            Mel jensen C. Iyoy